### PR TITLE
Fixing logs.md

### PR DESCRIPTION
**Pull request by fgalan**
FIX fixing logs.md documentation to align it with changes regarding #1582, #1585 and #1697 (along with small fixes in the code)

@kzangeli 
@crbrox

### CODE REVIEW

File: doc/manuals/admin/logs.md [hunk 1]
```diff
@@ -108,52 +110,76 @@ The different fields in each line are as follows:
...
     (this information is useful mainly for Orion developers).
 
-[Top](#top)
-    
-## Error and warning types
+[Top](#top)  
+
+## Alarms
+
+Alarm conditions:
```
Comment (fgalan) [inline doc/manuals/admin/logs.md]: This table is very hard to see in the diff view. I'd recomend to have a look to it in the "View" buttom.

File: doc/manuals/admin/logs.md [hunk 2]
```diff
@@ -108,52 +110,76 @@ The different fields in each line are as follows:
...
+| 2          | CRITICAL   | The following ERROR text appears in the 'msg' field: "Runtime Error (`<detail>`)"                                 | N/A                                                                                                                                                                                                                                       | Runtime Error. The `<detail>` text containts the detailed information.          | Restart Orion Context Broker. If it persists (e.g. new Runtime Errors appear within the next hour), scale up the problem to development team.
```
Comment (fgalan) [inline doc/manuals/admin/logs.md]: Not sure about "CRITICAL" for "Runtime Error". Currently, they correspond to things that it shouldn't happend but Orion uses to be quite protected agains crash due to them. However, they are still critical in the sense that developers team should be reported about that in order to have a look...

File: src/lib/rest/httpRequestSend.cpp [hunk 3]
```diff
@@ -436,8 +436,8 @@ int httpRequestSendWithCurl
     //
     // NOTE: This log line is used by the functional tests in cases/880_timeout_for_forward_and_notifications/
     //       So, this line should not be removed/altered, at least not without also modifying the functests.
-    //
-    LM_W(("Notification failure for %s:%s (curl_easy_perform failed: %s)", ip.c_str(), portAsString, curl_easy_strerror(res)));
+    //    
+    alarmMgr.notificationError(url, "(curl_easy_perform failed: " + std::string(curl_easy_strerror(res)) + ")");
```
Comment (kzangeli) [inline src/lib/rest/httpRequestSend.cpp]: Does this work?

Normally, the 'std::string' would be on the **first** string of the addition.
The second string should not need it as std::string has a 'plus-operator' accepting char* ...

Comment (fgalan) [inline src/lib/rest/httpRequestSend.cpp]: As far as I have checked, it works.

However, maybe I'm wrong... let jenkins decide ;)

NTC

**Comment by kzangeli** [comment]
LGTM, all yours @crbrox

**Comment by crbrox** [comment]
LGTM

**Pull request merged**